mos.yml

```diff
@@ -38,6 +38,10 @@ config_schema:
   - ["shelly.overheat_on", "i", 100, {title: "Overheat protection mode kicks in at or above this temperature"}]
   - ["shelly.overheat_off", "i", 90, {title: "Overheat protection mode turns off when the temperature is back below this threshold"}]
 
+  - ["se", "o", {title: "Sensor settings", abstract: true}]
+  - ["se.name", "s", "", {title: "Name of the sensor"}]
+  - ["se.unit", "i", 0, {title: "Display unit of the Sensor"}] # is dependant on the sensor type, for temperature: 0 - Celsius, 1 - Farenheit, other sensors 0 - default
```

Comment:
this is only about display, right? HAP only specifies celsius.
let's remove the setting, we can present both on the web ui i.e. "Value: 10°C (50°F)"

Reply:
> HAP only specifies celsius.

That's not fully correct, the "Current Temp" is only celsius but HAP specifies a "Temp Display Unit" which can be C or F.

Here a example of the Home Accessory Simulator and the Home+ App:
<img width="766" alt="Screenshot 2022-01-04 at 17 57 32" src="https://user-images.githubusercontent.com/165599/148095247-57aba3f3-6d2a-4315-9662-320bc55a5ef2.png">
![image](https://user-images.githubusercontent.com/165599/148095308-7dc5556d-d64d-4521-a616-9711ea724b90.png)

So I think it make sense to support it.

Reply:
i see that there is a characteristic for display units but temperature sensor service does not define it:

Required Characteristics
”9.35 Current Temperature” (page 175)
Optional Characteristics
”9.62 Name” (page 188)
”9.96 Status Active” (page 212)
”9.97 Status Fault” (page 212)
”9.99 Status Low Battery” (page 213)
”9.100 Status Tampered” (page 214)

units characteristic is read/write, it makes sense for something that actually displays the temperature, which shelyl does not.

Reply:
I was also confused yesterday, as it is not defined for the Temperature Sensor as you state. However TemperatureValue explicitly states it as inherent optional property. I guess that is an inconsistency in the spec. But the Simulator also defines it as @timoschilling shows in the screens above

Reply:
ok, fair enough.